docs/deploy/providers/heroku.md

```diff
@@ -23,3 +23,11 @@ Nitro supports deploying on [Heroku](https://heroku.com/) with minimal configura
    ```bash
    heroku config:set NITRO_PRESET=heroku
    ```
+
+1. Ensure you have `start` and `build` commands in your `package.json` file.
+
+   ```
```

Comment:
```suggestion
   ```json5
```

Reply:
package.json does not allow trailing commas and comments. it can be nice for documentation purpose but also tricky for lazy copy pasters